src/models/healthCertV2.test.ts

```diff
@@ -105,16 +105,15 @@ describe("src/models/healthCertV2", () => {
             patientName: "Tan Chen Chen",
             nationality: "Singaporean",
             gender: "She",
-            performerMcr: "123214",
+            performerMcr: "123456",
             performerName: "Dr Michael Lim",
             provider: "MacRitchie Medical Clinic",
             swabCollectionDate,
             swabType: "Anterior nares swab",
             swabTypeCode: "697989009",
             testResult: "Negative",
             testResultCode: "260385009",
-            testType:
-              "SARS-CoV-2 (COVID-19) Ag [Presence] in Upper respiratory specimen by Rapid immunoassay",
+            testType: "REAL TIME RT-PCR SWAB",
```

Comment:
This is the part I'm unsure about. Shouldn't we expect the `testType` = `Observation.valueCodeableConcept.coding[0].display`?

PCR: `"SARS-CoV-2 (COVID-19) RNA panel - Respiratory specimen by NAA with probe detection"`
ART: `"SARS-CoV-2 (COVID-19) Ag [Presence] in Upper respiratory specimen by Rapid immunoassay"`

If `"REAL TIME RT-PCR SWAB"` is only specific to EU QR, should we [set it here](https://github.com/Notarise-gov-sg/api-notarise-healthcerts/blob/master/src/models/euHealthCert/createEuHealthCert/createEuTestCert.ts#L50) instead?

Reply:
Since this is an unrelated topic, will approve the PR first.